### PR TITLE
New module for Context Broker API v2

### PR DESCRIPTION
**Pull request by vramperez**

### CODE REVIEW

File: orion_context_broker/orionModule_v2.js [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+                                status: resp.statusCode,
+                                body: body,
+                            }
+                            return callback(err, response);
+                        });
+                    }
+                });
+            });
+        }
+    });
```
Comment (fdelavega) [inline orion_context_broker/orionModule_v2.js:282]: You have some indentation issues in this file

**Comment by vramperez** [comment]
New commit fixing identation issues

File: test/data.js [hunk 2]
```diff
@@ -218,7 +228,7 @@ data.newEntityResp = {
     "type": ""
 };
 
-data.createSubscriptionReq = {
+data.createSubscriptionReq_v1 = {
```
Comment (fdelavega) [inline test/data.js]: Camel case var name

File: test/data.js [hunk 3]
```diff
@@ -242,7 +252,7 @@ data.createSubscriptionReq = {
     "throttling": "PT5S"
 };
 
-data.createSubscriptionResp = {
+data.createSubscriptionResp_v1 = {
```
Comment (fdelavega) [inline test/data.js]: camel case

File: test/data.js [hunk 4]
```diff
@@ -273,4 +299,35 @@ data.cancelSubscriptionResp = {
     "subscriptionId": data.DEFAULT_SUBS_ID
 };
 
+data.createSubscriptionReq_v2 = {
```
Comment (fdelavega) [inline test/data.js]: camel case

File: test/integration/accounting_CB_spec.js [hunk 5]
```diff
@@ -164,6 +192,130 @@ describe('Testing the accounting API. Orion Context-Broker requests', function (
...
+            payload = JSON.stringify(data.updateNotificationUrl);
+        }
+
+        if (expirationDateBefore === true) {
+            payload = JSON.stringify(data.updateExpirationDateBefore);
+        }
+
+        if (expirationDateBefore === false) {
+            payload = JSON.stringify(data.updateExpirationDateAfter);
+        }
```
Comment (fdelavega) [inline test/integration/accounting_CB_spec.js:278]: Having two ifs checking the true and false conditions is quite weird

File: test/unit/orionModule_v2_spec.js [hunk 6]
```diff
@@ -0,0 +1,519 @@
+var assert = require('assert'),
```
Comment (fdelavega) [inline test/unit/orionModule_v2_spec.js:1]: Review the indentation of this file

**Comment by vramperez** [comment]
New commits

**Pull request merged**